int32 n_readings
float32[] ranges
float32 start_angle
float32 FOV
float32 angle_increment
float32 max_range
